docs/content/guides/author-apps/kubernetes/how-to-kubernetes-resource/snippets/app-kubernetes.bicep

```diff
@@ -0,0 +1,44 @@
+// Import the set of Radius resources (Applications.*) into Bicep
+import radius as radius
+
+//KUBERNETES
+@description('Specifies Kubernetes namespace for the user.')
+param namespace string = 'default'
```

Comment:
Is this the same namespace that the application's container is deployed to? If not, how can we ensure that we deploy to the same namespace as the application's containers?